### PR TITLE
callback fn

### PR DESCRIPTION
**Pull request by alex461919-test**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -8,22 +8,19 @@ import { login_request } from "./store/authSlice";
 
 function App() {
   const dispatch = useAppDispatch();
+  const onCLick = useCallback(
+    (event) => {
+      dispatch(login_request({ user: "user", password: "password" }));
+    },
+    [dispatch]
+  );
```
Comment (alex461919) [inline src/App.tsx:17]: sure?
```suggestion
A you sure?
```

**Review by alex461919 — COMMENTED** [review]
Total

File: src/App.tsx (hunk 1 above)
Comment (alex461919-test) [reply src/App.tsx:17]: yes!

Comment (alex461919) [reply src/App.tsx:17]: talk1

Comment (alex461919-test) [reply src/App.tsx:17]: ok1

Comment (alex461919-test) [reply src/App.tsx:17]: ok2

**Review by alex461919 — APPROVED** [review]

**Pull request merged**